package.json

```diff
@@ -18,8 +18,9 @@
 	},
 	"scripts": {
 		"watch": "yarn install-if-deps-outdated && ./node_modules/.bin/gulp watch",
-		"clean": "yarn clean-client && yarn clean-extensions",
+		"clean": "yarn clean-client && yarn clean-extensions && yarn clean-composer",
 		"clean-client": "rm -rf _inc/build/ css/",
+		"clean-composer": "rm -rf composer.lock vendor/",
```

Comment:
Question: should this also clean the `composer.lock`s inside each package? `rm -rf packages/**/composer.lock`

Reply:
I don't know :)

Reply:
It will affect only things that run the package as a root package (we only do that for `composer phpunit`), so we should be safe to do it, or not. Let's go without it for now, I guess, we can always add it later 😉